packages/front-end/components/Forms/Field.tsx

```diff
@@ -95,6 +97,8 @@ const Field = forwardRef(
       optionGroups,
       type = "text",
       initialOption,
+      comboBox,
```

Comment:
If you google "HTML combo box" the results will confirm that a combo box is a standardized naming convention for this type of Filed comp. I was unaware that it was a UI pattern myself. I believe it is called a combobox because it combines an input field and an options dropdown comp. I feel like `allowNewOptions` is a misleading name because we are not allowing new options we are allowing the choice of options or input. So if we are to go forward with a new name I propose `optionsOrInputBox`, `inputOptionsBox`, or `optionsAndInputBox`.

Reply:
ok, I'm fine with `comboBox` as the prop name